### PR TITLE
List groups for user

### PR DESCRIPTION
**Pull request by jasebell**
We provide a route so users can see a list or groups available to them

### CODE REVIEW

File: src/witan/httpapi/queries.clj [hunk 1]
```diff
@@ -17,6 +18,20 @@
   [{:keys [kixi.datastore.metadatastore/type]}]
   (= type "stored"))
 
+(defn get-groups [requester user]
+  (let [[s r] (requests/GET requester
+                            :heimdall
+                            "/groups/search"
```
Comment (tcoupland) [inline src/witan/httpapi/queries.clj]: No paging?

File: src/witan/httpapi/api.clj [hunk 2]
```diff
@@ -179,10 +184,10 @@
...
           :return ::s/receipt-id-container
           (let [[s r headers] (activities/update-metadata!
-                               (activities req)
-                               (user req)
-                               metadata-updates
-                               id)]
+                                (activities req)
+                                (user req)
+                                metadata-updates
+                                id)]
```
Comment (tcoupland) [inline src/witan/httpapi/api.clj:192]: Uho, looks like some formatting rules are changing... is everyone just going to back an forth on this stuff from now?

File: src/witan/httpapi/queries.clj (hunk 1 above)
Comment (acron0) [reply src/witan/httpapi/queries.clj]: No, heimdall doesn't support paging.

File: src/witan/httpapi/api.clj (hunk 2 above)
Comment (acron0) [reply src/witan/httpapi/api.clj:192]: ¯\\_(ツ)_/¯ It doesn't really bother me. Any discussion about indentation vs. editors vs. style guides is basically just one big bike shed.

Comment (mattford63) [reply src/witan/httpapi/api.clj:192]: ?w=1

**Review by otfrom — APPROVED** [review]

**Pull request merged**